includes/events.js

```diff
@@ -1,63 +1,54 @@
 module.exports = (params) => {
-  var namespaceFilterSql = '';
-  if (params.bqEventsTableNameSpace) {
-    namespaceFilterSql = `AND namespace = '${params.bqEventsTableNameSpace}'`;
-  }
-  return publish("events_" + params.eventSourceName, {
```

Comment:
The only change to this part is extra metadata on the DATA field (line 35/31) - the rest is just autoformatted indentation changes